vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb

```diff
@@ -0,0 +1,22 @@
+module SecureRooms
+
+  class FacilityOccupanciesController < ApplicationController
+
+    include TabCountHelper
```

Comment:
Are we using this helper anywhere yet? If you know for sure already we're going to be using it, I'm fine leaving it in here.